eng/Versions.props

```diff
@@ -123,8 +125,9 @@
     <MonoDevelopSdkPackageVersion>1.0.15</MonoDevelopSdkPackageVersion>
     <MoqPackageVersion>4.10.0</MoqPackageVersion>
     <!-- STOP!!! We need to reference the version of JSON that our HOSTS supprt. -->
-    <NewtonsoftJsonPackageVersion>9.0.1</NewtonsoftJsonPackageVersion>
-    <OmniSharpExtensionsLanguageServerPackageVersion>0.14.2</OmniSharpExtensionsLanguageServerPackageVersion>
+    <NewtonsoftJsonPackageVersion>11.0.2</NewtonsoftJsonPackageVersion>
```

Comment:
Could you verify that this in VS by booting VS (not by hitting f5 in our solution), attaching and seeing the version of Newtonsoft.Json loaded?

@KirillOsenkov what version of Newtonsoft.Json is VS4Mac on?

Reply:
We're on 12.0.2. We have a binding redirect in place so we should be good with 11.0.2 here.

Reply:
If ya'll are using 12.0.2 I may as well do so too.

Reply:
> If ya'll are using 12.0.2 I may as well do so too.

Gotta make sure it's in VS Windows though too